README.md

```diff
@@ -13,9 +13,12 @@ Aka IFTTT orchestration for Ops.
 * Ubuntu Xenial (16.04)
 * RHEL6 / CentOS6
 * RHEL7 / CentOS7
+* RHEL8 / CentOS8
 
 > If you're using the provided Vagrantfile, note that it uses Xenial by default.
 
+> If you're installing on RHEL8 / CentOS8 and firewalld is enabled, then you must ensure that the http and https services are enabled. The ansible-st2 roles do not configure the firewalld rules.
```

Comment:
An improvement to this would be having a reminder to users mentioning firewall rules they need to manage in order to open st2web access. The reason is that EL8 and very specific firewall tool is not the case here. Same could be applied to any OS with whatever firewall system it runs. 

For example: 
> In order to access StackStorm Web UI, please don't forget to ensure that `http`/`https` ports are opened in your firewall system.

And so this could be applied to any OS, any firewall system is it internal or cloud-layer firewall.